pkg/nuclide/format-js-base/lib/constants/builtIns.js

```diff
@@ -17,24 +17,51 @@ module.exports = new Set([
   'Boolean',
   'Date',
   'Error',
+  'Function',
+  'Infinity',
   'JSON',
   'Math',
   'Number',
   'Object',
+  'Promise',
   'RegExp',
   'String',
   'Symbol',
+
   'arguments',
   'global',
+  'isFinite',
+  'isNaN',
   'parseFloat',
   'parseInt',
   'undefined',
   'console',
 
   // Browser built ins.
+  'alert',
+  'clearInterval',
+  'clearTimeout',
+  'confirm',
+  'decodeURI',
+  'decodeURIComponent',
   'document',
+  'encodeURI',
+  'encodeURIComponent',
+  'escape',
+  'indexedDB',
   'location',
+  'localStorage',
+  'open',
+  'performance',
+  'prompt',
+  'screen',
+  'sessionStorage',
+  'setInterval',
+  'setTimeout',
   'window',
+  
+  'Option',
+  'Image',
```

Comment:
I think this one will conflict with `Image.react` which we do want to require. Can we remove this?